test/elpy-insert--pip-button-value-create-test.el

```diff
@@ -12,8 +12,10 @@
     (mletf* ((executable-find
               (command)
               (cond
-                ((equal command "pip") t)
                 ((equal command "easy_install") t)))
+             (call-process
+              (program &optional infile destination display &rest args)
+               0)
              (widget (widget-create 'elpy-insert--pip-button "test-module")))
       (should (string-match "pip" (widget-get widget :command)))
       (should (string-match "pip" (buffer-string))))))
```

Comment:
I do not understand that argument. Why is it unnecessary to test whether the code actually calls the command you want it to call? The whole point of this patch is to change from "pip" to "python -m pip", no?